### PR TITLE
QT1-440 file upload add log and schema and db 

### PR DESCRIPTION
**Pull request by alireid**

### CODE REVIEW

File: src/Approvals/SFA.DAS.Approvals.Api/Controllers/BulkUploadController.cs [hunk 1]
```diff
@@ -76,5 +77,26 @@ public async Task<IActionResult> AddDraftapprenticeships(BulkUploadAddDraftAppre
...
+                    FileName = request.FileName,
+                    RplCount = request.RplCount,
+                    RowCount = request.RowCount,
+                    FileContent = request.FileContent
+                });
+
+            if (result == null)
+            {
+                return NotFound();
+            }
```
Comment (Pauljgraham) [inline src/Approvals/SFA.DAS.Approvals.Api/Controllers/BulkUploadController.cs:98]: Do we need the not found result?  We don't have a test covering it and it shouldn't happen. Should it?

File: src/Approvals/SFA.DAS.Approvals.Api/Models/BulkUploadAddLogRequest.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿namespace SFA.DAS.Approvals.Api.Models
+{
+    public class BulkUploadAddLogRequest
+    {
+        public BulkUploadAddLogRequest()
+        {
+        }
```
Comment (Pauljgraham) [inline src/Approvals/SFA.DAS.Approvals.Api/Models/BulkUploadAddLogRequest.cs:7]: The convention is we usually don't add ctors if they are not doing anything

File: src/Approvals/SFA.DAS.Approvals/Application/BulkUpload/Commands/BulkUploadAddLogCommand.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
+﻿using MediatR;
+
+namespace SFA.DAS.Approvals.Application.BulkUpload.Commands
+{
+    public class BulkUploadAddLogCommand : IRequest<BulkUploadAddLogResult>
+    {
+        public BulkUploadAddLogCommand()
+        {
+        }
+
```
Comment (Pauljgraham) [inline src/Approvals/SFA.DAS.Approvals/Application/BulkUpload/Commands/BulkUploadAddLogCommand.cs:10]: The convention is we usually don't add ctors if they are not doing anything

File: src/Approvals/SFA.DAS.Approvals/Application/BulkUpload/Commands/BulkUploadAddLogResult.cs [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+        public int LogId { get; set; }
+
+        public static implicit operator BulkUploadAddLogResult(InnerApi.Responses.BulkUploadAddLogResponse response)
+        {
+            return new BulkUploadAddLogResponse
+            {
+                LogId = response.LogId
+            };
+        }
```
Comment (Pauljgraham) [inline src/Approvals/SFA.DAS.Approvals/Application/BulkUpload/Commands/BulkUploadAddLogResult.cs:17]: IS this code called. It's lacking a test (not I worry too much about that in this case), but I can't see where it is being used

File: src/Approvals/SFA.DAS.Approvals/InnerApi/Responses/BulkUploadAddLogResponse.cs [hunk 5]
```diff
@@ -0,0 +1,10 @@
+﻿namespace SFA.DAS.Approvals.InnerApi.Responses
+{
+    public class BulkUploadAddLogResponse
+    {
+        public BulkUploadAddLogResponse() { }
+
```
Comment (Pauljgraham) [inline src/Approvals/SFA.DAS.Approvals/InnerApi/Responses/BulkUploadAddLogResponse.cs:6]: The convention is we usually don't add ctors if they are not doing anything

**Review by Pauljgraham — COMMENTED** [review]
A few very minor comments

File: src/Approvals/SFA.DAS.Approvals.Api/Controllers/BulkUploadController.cs (hunk 1 above)
Comment (alireid) [reply src/Approvals/SFA.DAS.Approvals.Api/Controllers/BulkUploadController.cs:98]: Good shout. Removed

File: src/Approvals/SFA.DAS.Approvals/Application/BulkUpload/Commands/BulkUploadAddLogCommand.cs (hunk 3 above)
Comment (alireid) [reply src/Approvals/SFA.DAS.Approvals/Application/BulkUpload/Commands/BulkUploadAddLogCommand.cs:10]: Removed

File: src/Approvals/SFA.DAS.Approvals/InnerApi/Responses/BulkUploadAddLogResponse.cs (hunk 5 above)
Comment (alireid) [reply src/Approvals/SFA.DAS.Approvals/InnerApi/Responses/BulkUploadAddLogResponse.cs:6]: Removed

File: src/Approvals/SFA.DAS.Approvals/Application/BulkUpload/Commands/BulkUploadAddLogResult.cs (hunk 4 above)
Comment (alireid) [reply src/Approvals/SFA.DAS.Approvals/Application/BulkUpload/Commands/BulkUploadAddLogResult.cs:17]: Hey @Pauljgraham. Yes, youre right, was superfluous code there. Cleaned up it wasn't used.

**Comment by alireid** [comment]
> A few very minor comments

@Pauljgraham ready for recheck, thanks

**Review by Pauljgraham — DISMISSED** [review]

**Review by Pauljgraham — DISMISSED** [review]

**Pull request closed without merging**